logic/transaction_pool.js

```diff
@@ -975,13 +990,34 @@ __private.applyUnconfirmedList = function(transactions, cb, tx) {
  */
 __private.transactionTimeOut = function(transaction) {
 	if (transaction.type === transactionTypes.MULTI) {
-		return transaction.asset.multisignature.lifetime * 3600;
+		return (
+			transaction.asset.multisignature.lifetime *
+			library.config.transactions.multiSignatureTimeoutMultiplier
+		);
 	} else if (Array.isArray(transaction.signatures)) {
-		return constants.unconfirmedTransactionTimeOut * 8;
+		return (
+			constants.unconfirmedTransactionTimeOut *
+			library.config.transactions.signaturesTimeoutMultiplier
+		);
 	}
 	return constants.unconfirmedTransactionTimeOut;
 };
 
+/**
+ * Check if transaction is expired.
+ *
+ * @private
+ * @param {Object} transactions - transaction to be expired
+ * @returns {Boolean} true if transactions expired
+ */
+__private.isExpired = transaction => {
+	const timeNow = Math.floor(Date.now() / 1000);
+	const timeOut = __private.transactionTimeOut(transaction);
+	// transaction.receivedAt is instance of Date
+	const seconds = timeNow - Math.floor(transaction.receivedAt.getTime() / 1000);
+	return seconds > timeOut;
```

Comment:
Please rename `seconds` to `time Passed`, that will make it more readable `return timePassed > timeOut`